docs/docs/recipes/create-your-connector/fragments/_index_contents.md

```diff
@@ -0,0 +1,20 @@
+Connector's class is an implementation among _SMSConnector_, _EmailConnector_, and _SocialConnector_. See (TODO: add a link to _@logto/connector-types_) to build a picture of connector types and connector design.
```

Comment:
I think you can just show the base class definitions here since they are simple.